src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java

```diff
@@ -173,6 +179,18 @@ private static StructureWrapper loadSchematic(
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)